src/signaling.ts

```diff
@@ -34,10 +35,10 @@ class Signaling extends EventEmitter {
         window.addEventListener("unload", (event) => {
           this.disconnect();
         });
-        await this.setMediaPreferences();
-        this.pingInterval = setInterval(() => {
```

Comment:
Sorry for another dumb question... Is there a reason why we removed this setInterval() call?

Reply:
Not a dumb question. I removed it for testing and didn't realize I had removed it in the PR. Thank you for catching!